.markdownlint.jsonc

```diff
@@ -39,5 +39,10 @@
       "markdownlint",
       "Node.js"
     ]
+  },
+
+  // MD-46/code-block-style
+  "MD046": {
+    "style": "fenced"
```

Comment:
We use fenced block style all the time (backticked code blocks). `markdownlint` by default expects `indented` code blocks. I added the rule to avoid pre-commit to complain and fail.